frontend/src/routes/(authenticated)/admin/ProjectTable.svelte

```diff
@@ -177,6 +191,9 @@ async function softDeleteProject(project: Project): Promise<void> {
                     {$t('admin_dashboard.column_last_change')}
                     <span class="i-mdi-sort-ascending text-xl align-[-5px] ml-2"/>
                 </th>
+                <AdminContent>
```

Comment:
This is the admin page, so I don't think we need this here or on 227 🙂

Reply:
Done.